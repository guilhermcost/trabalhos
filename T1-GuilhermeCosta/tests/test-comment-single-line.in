this is a// comment really 
this is b

