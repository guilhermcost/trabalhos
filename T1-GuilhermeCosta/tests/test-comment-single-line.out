(1,ID,"this")
(1,ID,"is")
(1,ID,"a")
(2,ID,"this")
(2,ID,"is")
(2,ID,"b")
